### PR TITLE
Menu

### PR DESCRIPTION
**Pull request by oneill-c**
**Depends on**
N/A

**What's included**
- [x] Add menu system to project

### CODE REVIEW

File: src/components/menu/menu.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    },
+    {
+      key: 'recoil-example-page-1',
+      text: 'Recoil Example',
+      navigateTo: '/recoil-example-page-1',
+    },
+  ];
+
+  return (
+    <div>
```
Comment (iGary) [inline src/components/menu/menu.tsx]: Think the root div isn't needed

File: src/components/menu/menu.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      <div className="flex space-x-4">
+        {navItems.map((item: NavigationItem) => (
+          <MenuItem
+            id={item.key}
+            active={item.navigateTo === pathname}
+            navigateTo={item.navigateTo}
+            key={item.key}
+            text={item.text}
+            // icon={<item.icon size={16} />}
```
Comment (iGary) [inline src/components/menu/menu.tsx]: Can be removed

**Review by iGary — APPROVED** [review]
LGTM, two things not blockers though

File: src/components/menu/menu.tsx (hunk 2 above)
Comment (oneill-c) [reply src/components/menu/menu.tsx]: I left this in here as a wee example of how to add an icon

**Pull request merged**